src/UserStories/UserStories.css

```diff
@@ -8,31 +8,36 @@ html {
   overflow: hidden;
 }
 .stories-scrollable {
+  transition: transform 0.2s ease-out;
+  min-height: 100vh;
+  flex-wrap: nowrap;
   display: flex;
-  padding-left: 50vw;
-  transition: transform 0.5s ease-in-out;
-  height: 100vh;
+  align-items: center;
 }
 .stories-container {
-  margin: 70px auto;
-  height: 800px;
-  min-width: 449px;
+  margin: 18px 0;
   border-radius: 8px;
-  margin-left: 50px;
+  transition: all 0.5s ease-out;
+  flex: 0 0 auto;
+}
+.stories-container.is-expanded {
+  border-left: 46px solid transparent;
+  border-right: 46px solid transparent;
+}
+.stories-container.is-expanded:hover *.next-prev-buttons {
+  opacity: 1;
 }
 .stories-container.is-small {
   transform: scale(0.4);
 }
 .instagram-logo-link {
   position: absolute;
-  left: 20px;
-  top: 10px;
+  left: 16px;
+  top: 15px;
   font-size: 2em;
-  color: white;
   text-decoration: none;
-  font-family: 'Festive', cursive !important;
   margin: 0;
-  z-index: 2;
+  z-index: 20;
```

Comment:
Do you need a z-index? I don't think so.

Reply:
Not since we switched back to scaling both items.